qase-python-commons/src/qase/commons/profilers/network.py

```diff
@@ -47,15 +47,17 @@ def wrapper(self, request, *args, **kwargs):
     def _urllib3_request_wrapper(self, func):
         @wraps(func)
         def wrapper(self, method, url, *args, **kwargs):
+            if 'qase.io' in url:
+                return func(self, method, url, *args, **kwargs)
+
```

Comment:
So, the profiler won't profile requests to qase.io, made for creating test runs and reporting results, right?

Users can use a proxy or custom domain for Qase app. Let's bind this to the Qase URL config variable.

Reply:
fixed